dev/breeze/src/airflow_breeze/commands/testing_commands_config.py

```diff
@@ -96,6 +96,7 @@
                 "--mount-sources",
                 "--skip-docker-compose-down",
                 "--skip-providers",
+                "--keep-env-variables",
```

Comment:
this should also be added in few other places where similar flags are passed - for example in `breeze shell` command and also in `db-tests` and `non-db-tests`. This way we are able to just enter breeze (with `breeze shell` and manually run testing and iterate on it and run db/non-db tests separately.

Reply:
Done